nearlib/signing/account_info.js

```diff
@@ -0,0 +1,79 @@
+const KeyPair = require('./key_pair');
+
+/**
+ * Standard format for storing account info in json
+ */
+class AccountInfo {
+
+    constructor(accountId, keyPair, networkId) {
+        this.accountId = accountId;
+        this.keyPair = keyPair;
+        this.networkId = networkId;
+    }
+
+    /**
+     * Reconstruct account info object from json.
+     * @param {Object} json 
+     */
+    static fromJson(json) {
+        if (!json.public_key || !json.secret_key || !json.account_id || !json.network_id) {
+            throw 'Invalid account info format. Please ensure it contains public_key, secret_key, and account_id".';
+        }
+        return new AccountInfo(json.account_id, new KeyPair(json.public_key, json.secret_key), json.network_id);
+    }
+
+    /**
+     * Convert to standard json format.
+     */
+    toJson() {
+        return {
+            account_id: this.accountId,
+            public_key: this.keyPair.getPublicKey(),
+            secret_key: this.keyPair.getSecretKey(),
+            network_id: this.networkId
+        };
+    }
+
+    /**
+     * Gets a key pair from account info.
+     */
+    getKeyPair() {
+        return this.keyPair;
+    }
+
+    /**
+     * Gets a key pair from account info.
+     */
+    getAccountId() {
+        return this.accountId;
+    }
+
+    getNetworkId() {
+        return this.networkId;
+    }
+
+    /**
+     * Utility function to download account info as a standard file.
+     */
+    downloadAsFile() {
+        const fileName = getKeyFileName();
+        const text = JSON.stringify(this.toJson());
+      
+        var element = document.createElement('a');
```

Comment:
I don't think code depending on `document` (i.e. browser-only) should live in a module that is included in Node version as well.

Reply:
Indeed seems like this belongs to frontend code.

Reply:
I am on the fence about this. 
pros: 
* multiple projects will want to use this (studio, wallet, etc.) - they don't have to duplicate the code
* we have dependency on browser stuff in browserkeystore, so if we want to have no browser dependencies in nearlib we need to refactor already -- what's the benefit? Also where would these common dependencies go --- are we going to maintain a new library with frontend-only common code? 

cons:
already stated

Reply:
yeah, we should also:
* remove browser-specific stuff from `index.js`
* set separate entry point for browser-specific stuff using https://docs.npmjs.com/files/package.json#browser

Given how many browser special case hacks is in general purpose module for this (https://github.com/eligrey/FileSaver.js/blob/master/src/FileSaver.js) – I think it might be best to have frontend code use it instead.